drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/downloading/source/ExternalBinarySource.java

```diff
@@ -40,4 +59,17 @@
      * @return A regex that represents a file name of an asset to be downloaded.
      */
     String getFileNameRegexToDownload(String version);
+
+    /**
+     * This method should returns a regex that represents a file name of the release asset that should be downloaded.
+     * If this method returns null, the AbstractBinaryHandler won't be looking for any file in the cache directory
+     * and will jump to downloading phase.
```

Comment:
I'm not convinced we should document any behaviour of anything implementing that interface. I would simply remove it.